docs/resources/servergroup.md

```diff
@@ -12,29 +11,139 @@ Represent server group resource
+  name       = "my-servergroup"
+  policy     = "affinity"
+  project_id = data.gcore_project.project.id
+  region_id  = data.gcore_region.region.id
+}
+```
+
+### Creating server group with two instances
+
+```terraform
+data "gcore_image" "ubuntu" {
+  name       = "ubuntu-22.04-x64"
+  region_id  = data.gcore_region.region.id
+  project_id = data.gcore_project.project.id
+}
+
+resource "gcore_volume" "boot_volume" {
+  name       = "my-boot-volume"
+  type_name  = "ssd_hiiops"
+  size       = 5
+  image_id   = data.gcore_image.ubuntu.id
+  project_id = data.gcore_project.project.id
+  region_id  = data.gcore_region.region.id
+}
+
+resource "gcore_volume" "boot_volume2" {
+  name       = "my-boot-volume2"
+  type_name  = "ssd_hiiops"
+  size       = 5
+  image_id   = data.gcore_image.ubuntu.id
+  project_id = data.gcore_project.project.id
+  region_id  = data.gcore_region.region.id
+}
+
+resource "gcore_instancev2" "instance" {
+  flavor_id     = "g1-standard-2-4"
+  name          = "my-instance"
+
+  volume {
+    volume_id  = gcore_volume.boot_volume.id
+    boot_index = 0
+  }
+
+  interface {
+    type = "external"
+    name = "my-external-interface"
+  }
+
+  project_id = data.gcore_project.project.id
+  region_id  = data.gcore_region.region.id
+}
+
+resource "gcore_instancev2" "instance2" {
+  flavor_id     = "g1-standard-2-4"
+  name          = "my-instance2"
+
+  volume {
+    volume_id  = gcore_volume.boot_volume.id
+    boot_index = 0
+  }
+
+  interface {
+    type = "external"
+    name = "my-external-interface"
+  }
+
+  project_id = data.gcore_project.project.id
+  region_id  = data.gcore_region.region.id
+}
+
+resource "gcore_servergroup" "servergroup" {
   name       = "default"
   policy     = "affinity"
-  region_id  = 1
-  project_id = 1
+  project_id = data.gcore_project.project.id
+  region_id  = data.gcore_region.region.id
+  instance {
+      instance_id = gcore_instancev2.instance.id
+  }
+
+  instance {
+      instance_id = gcore_instancev2.instance2.id
+  }
 }
 ```
 
+
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
 ### Required
 
 - `name` (String) Displayed server group name
-- `policy` (String) Server group policy. Available value is 'affinity', 'anti-affinity'
+- `policy` (String) Server group policy. Available value is 'affinity' 'anti-affinity' 'soft-anti-affinity'. 
+
+**Affinity rule**:
+
+Boost performance by grouping related instances together. They will be hosted on the same physical server for
+faster interaction, enhancing communication and resource sharing.
+
+**Anti-affinity soft rule**:
+
+Balance separation and flexibility, occasionally placing instances on different physical servers for fault tolerance,
+or on the same server physical server for faster interaction and resource optimization.
+
+**Anti-affinity**: 
+
+Strictly isolate instances to different physical servers, minimizing simultaneous failures for critical applications.
+Choose this for enhanced fault tolerance.
```

Comment:
anti-affinity: Restricts instances belonging to the server group to separate hosts.
you mixed up the descriptions of Anti-affinity and affinity